packages/pagebuilder/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@magento/pagebuilder",
```

Comment:
We might want to think of a naming convention for packages like this, while I love the simplicity of @magento/pagebuilder, I think it could cause some confusion on it's intent, even with the description.

Reply:
I know, but I was really into the simplicity of this.

Other plugin systems, like `eslint` and `babel`, rely on verbose package naming conventions to auto-detect the plugins. That annoys me and it forces a given package to plug in to one and only one thing. I chose the `pwa-studio` property in `package.json` so that package names could be more freely chosen.

Nevertheless, we have our own namespacing to think about. Suggestions?

Reply:
Yeah I love the simplicity too, if people get confused we'll deal with that later.

I dislike naming conventions but sometimes they're important, so I have no good suggestions.